test/jdk/java/nio/file/FileStore/Basic.java

```diff
@@ -22,7 +22,7 @@
  */
 
 /* @test
- * @bug 4313887 6873621 6979526 7006126 7020517 8264400
+ * @bug 4313887 6873621 6979526 7006126 7020517 8264400 8277159
```

Comment:
I don't think that the bug ID needs to be added here as this is a change to the test itself, only.

Reply:
Thanks for your comments! I've removed the bug ID.